docs/ad/api.md

```diff
@@ -522,65 +568,232 @@ Sample Input:
+              "boost": 1
+            }
+          },
+          "detection_interval": {
+            "period": {
+              "interval": 5,
+              "unit": "Minutes"
+            }
+          },
+          "last_update_time": 1574208127000
+        }
+      }
+    ]
+  }
+}
+```
+
+## Get detector stats
+
+Provides information about how the plugin is performing.
+
+#### Request
+
+```json
+GET _opendistro/_anomaly_detection/stats/
+GET _opendistro/_anomaly_detection/stats/
+GET _opendistro/_anomaly_detection/<nodeId>/stats/
+GET _opendistro/_anomaly_detection/<nodeId>/stats/<stat>
+GET _opendistro/_anomaly_detection/stats/<stat>
+```
+
+#### Sample response
+
+```json
+{
+  "_nodes": {
+    "total": 3,
+    "successful": 3,
+    "failed": 0
+  },
+  "cluster_name": "multi-node-run",
+  "anomaly_detectors_index_status": "green",
+  "detector_count": 1,
+  "models_checkpoint_index_status": "green",
+  "anomaly_results_index_status": "green",
+  "nodes": {
+    "IgWDUfzFRzW0FWAXM5FGJw": {
+      "ad_execute_request_count": 8,
+      "models": [
+        {
+          "detector_id": "asbPyG4BlFrvFHzloDO-",
+          "model_type": "rcf",
+          "model_id": "asbPyG4BlFrvFHzloDO-_model_rcf_0"
+        },
+        {
+          "detector_id": "asbPyG4BlFrvFHzloDO-",
+          "model_type": "threshold",
+          "model_id": "asbPyG4BlFrvFHzloDO-_model_threshold"
+        }
+      ],
+      "ad_execute_failure_count": 7
+    },
+    "y7YUQWukQEWOYbfdEq13hQ": {
+      "ad_execute_request_count": 0,
+      "ad_execute_failure_count": 0,
+      "models": []
+    },
+    "cDcGNsPoRAyRMlPP1m-vZw": {
+      "ad_execute_request_count": 0,
+      "ad_execute_failure_count": 0,
+      "models": [
+        {
+          "detector_id": "asbPyG4BlFrvFHzloDO-",
+          "model_type": "rcf",
+          "model_id": "asbPyG4BlFrvFHzloDO-_model_rcf_2"
+        },
+        {
+          "detector_id": "asbPyG4BlFrvFHzloDO-",
+          "model_type": "rcf",
+          "model_id": "asbPyG4BlFrvFHzloDO-_model_rcf_1"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Create monitor
+
+Create a monitor to set up alerts for the detector.
+
+#### Request
+
+```json
+POST /_opendistro/_alerting/monitors
+{
+    "type":"monitor",
+    "name":"ylwu-test-nab_art_daily_jumpsdown-monitor",
+    "enabled":true,
+    "schedule":{
+        "period":{
+            "interval":5,
+            "unit":"MINUTES"
+        }
+    },
+    "inputs":[
+        {
+            "anomaly_detector":{
+                "detector_id":"dAaHjG4B5Uh30OvgBjjc"
             }
-        ]
+        }
+    ],
+    "triggers":[
+        {
+            "name":"ad-trigger",
+            "severity":"1",
+            "condition":{
+                "script":{
+                    "source":"ctx.results[0].anomalyGrade > 0 && ctx.results[0].confidence > 0.7",
+                    "lang":"painless"
+                }
+            },
+            "actions":[
+
+            ]
+        }
+    ]
+}
```

Comment:
It seems monitor creation doesn't work on latest AD/Alerting code. I tried this request on test-domain(2 data nodes, ~40 detectors), and got error saying `unable to parse Input with name [anomaly_detector]: parser not found`

@ylwu-amzn @kaituo  can you confirm if monitor creation works for latest AD/Alerting?